### DISCUSSION TITLE
Unable to fork

### DISCUSSION THREAD

**Discussion by keeper-22** [community/community#113507] [New to GitHub]
When I click create a new fork, nothing happens.
![image](https://github.com/community/community/assets/164086754/a1936c50-3dac-43ee-bd3d-f7d338c1c45d)

It goes to the create a new fork page
![image](https://github.com/community/community/assets/164086754/620d74e8-04d1-4bbf-b8ca-d227c3f7461e)

After clicking the button, there is a progress bar that loads quickly across the top of the page but nothing happens. I am put back at the repo page and the repo is not in my acount.

I am banging my head against the wall trying to figure out what I am doing wrong. Obviously I am new to this and this is my first attempt to fork, but no success. I am sure I am missing something obvious, please help.

**Comment by WilmerTavo** [comment]
Hello. 
You can refer to GitHub's own documentation for guidance. Here's the link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo

If you need help, don't hesitate to ask.

**Comment by Leorev01** [comment]
Check Permissions: Ensure you're logged into the correct account with forking permissions.
Browser Issues:
Clear cache and cookies.
Try a different browser.
Disable extensions.
Network: Ensure a stable internet connection; disable VPN/firewall temporarily.
Repository Settings: Confirm that forking is allowed on the repository.
Inspect Console: Check for errors in the browser's developer console (F12).
Retry: Wait a few minutes and try again.
Contact Support: If the issue persists, reach out to platform support.

**Comment by wassupjay** [comment]
I see that your account is new, So thats the reason. You cant do anything until you  have some activity.

**Reply by wassupjay** [reply]
If this helps, kindly mark it as answer! thanks

**Comment by Sheikh-Anas-Tauseef** [comment]
GitHub doesn’t allow you to fork a repo you already own or fork it twice into the same account. Try using another account or create a GitHub organization, transfer the repo there, and then fork it back. That should work! 👍
